open_academy/demo/open_academy_course_demo.xml

```diff
@@ -0,0 +1,32 @@
+<odoo>
+    <data>
+        <record model="openacademy.course" id="coursef1">
```

Comment:
First ID then model. See the guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#254demo-records
```suggestion
        <record id="coursef1" model="openacademy.course" >
```

Please apply this suggestion in all records.